from vk_api.exceptions import *
from objects import CodeExceptions
import pydantic



